basics/installation/localhost.md

```diff
@@ -217,6 +217,7 @@ This problem may arise in case-insensitive file systems like MacOS due to a misc
 
 [getting-started-guide]: https://doc.prestashop.com/display/PS17/Getting+Started
 [system-requirements]: {{< ref "1.7/basics/installation/system-requirements" >}}
+[installing-prestashop]: {{< ref "1.7/themes/getting-started/setting-up-your-local-environment/#installing-prestashop" >}}
```

Comment:
```suggestion
[installing-prestashop]: {{< ref "1.7/themes/getting-started/setting-up-your-local-environment/" >}}
```
Let's modify this to this string so that hugo is not crashing